### DISCUSSION TITLE
Digitalocean droplet Unexpected Error after install and login

### DISCUSSION THREAD

**Discussion by Kainkainkain** [directus/directus#5693] [Archive]
### Unexpected Error
Wanted to try directus on a server, pressed this droplet:
https://marketplace.digitalocean.com/apps/directus

- Signed up to digitalocean
- Chose cheapest option for droplet to try it out
- Droplet installed
- SSH to digitalocean VM and logged in, followed instructions:
- Added an A record, as instructed
- waited for propogation
- Logged in with example.com login
- redirected to [domain]/admin/collections
- Custom error screen "Unexpected Error".

I thought mabye the db wasnt installed so I npx directus database install - but that returned a 'db already installed'. 
Console shows GET[domain]/relations?limit=-1 500 error.

Error screen displays: 
{
  "errors": [
    {
      "message": "select \"directus_relations\".\"id\", \"directus_relations\".\"many_collection\", \"directus_relations\".\"many_field\", \"directus_relations\".\"one_collection\", \"directus_relations\".\"one_field\", \"directus_relations\".\"one_collection_field\", \"directus_relations\".\"one_allowed_collections\", \"directus_relations\".\"junction_field\", \"directus_relations\".\"sort_field\", \"directus_relations\".\"one_deselect_action\", \"directus_relations\".\"many_primary\", \"directus_relations\".\"one_primary\" from \"directus_relations\" order by \"directus_relations\".\"id\" asc - column directus_relations.many_primary does not exist",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}

I've googled the error, searched this forum and haven't discovered anyone else with similar issues. 
Console shows: 
Hey! Interested in helping build this open-source data management platform?
If so, join our growing team of contributors at: https://directus.chat
app.c12101e3.js:1 🐰 Starting Directus...
app.c12101e3.js:1 🕓 Application Loaded: 474.656982421875 ms
app.c12101e3.js:1 ✨ Project Information
app.c12101e3.js:1 Version: v9.0.0-rc.66
app.c12101e3.js:1 Environment: production

Sorry if I have missed a step, but does anyone know what I might have done wrong?

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/5691

You didn't do anything wrong 🙂 We had a bit of a bumpy release yesterday 👍🏻 Fix coming right up

**Reply by Kainkainkain** [reply]
Ah awesome.
